### ISSUE TITLE
forget and walk back

### ISSUE THREAD

**Issue by madtomic** [mcmonkey4eva/Sentinel#216]
enemy target time: 100
chaserange RANGE
These don't seems to work.

**Comment by mcmonkey4eva** [comment]
What about them doesn't work?

**Comment by Formula350** [comment]
Yea the Target Time doesn't seem to do anything; however, that could be part of a larger problem that I'm about to submit. Since I wasn't going to specifically mention that command, I'll post what I've come across...

I selected the fully-configured Sentinel NPC, then ran `/sentinel targettime 10`
I don't know what the value is counted in since the Help doesn't say, but I figured 10 would be either 1/2 a second or 10 seconds and sufficient for testing.

Nothing seemed to happen... The NPC continued following the player it had been, even after using `/sentinel forgive`. I even tried `/sentinel forgive <username>` just in case that might work, but nothing. 

The `chaserange` problem I think is that they don't return to their original spawn point like he wants it to. It just stops where it is until a new target enters that range, at which time it'll resume the chase (if not configured with a ranged weapon). At least that's the behavior it was doing on one of my bosses (Sentinels), until I increased the range size for the room it spawns in, then it followed them no matter what. 
Point is though, it wouldn't return to any location, just stop. Which goes against what its help topic says of: `Changes the maximum distance an NPC will run before returning to base.`
